### DISCUSSION TITLE
Override default max-keys option

### DISCUSSION THREAD

**Discussion by pradeep8985** [iterate-ch/cyberduck#14562] [Q&A] [answered]
Hello All,

I am aware that the default max-keys are limited to 1000. Is there any way to override this option? I have tried to override by using the following in custom profile, but it doesn't work. Any leads are much appreciated... 

```
        <key>Properties</key>
        <array>
	       <string>max-keys=10000</string>
        </array>
```

**Comment by dkocher** [comment, marked as answer]
I suppose you are referring to the `max-keys` parameter in S3 [^1]. You can use the property `s3.listing.chunksize` [^2]  instead as in your example which applies to this parameter.

[^1]: https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectVersions.html#API_ListObjectVersions_RequestSyntax
[^2]: https://github.com/iterate-ch/cyberduck/blob/7b6fdd3271e2752abe1dcc39fc61d03ffa43c10d/defaults/src/main/resources/default.properties#L305

**Reply by pradeep8985** [reply]
Hi @dkocher, Perfect, it worked... Thank you..
